package-managers/nuget.md

```diff
@@ -89,24 +89,43 @@ To work with public feeds, we just leave the credentials empty and Rider will co
 
 ### NuGet CLI
 
-#### Public feed
+#### Private feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
-#### Private feed
+#### Public feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
 ### dotnet CLI
 
-#### Public feed
+In this section we are going to add our packages via the dotnet CLI. We assume that the dotnet CLI tools are installed. If this is not the case, our friends at Microsoft can help us with that via [this link](https://www.microsoft.com/net/core).
+
+#### Private feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
-#### Private feed
+#### Public feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
+When adding a package to our project via the `dotnet CLI` we can get a generated command from the package details in our feed.
+
+![dotnet add command](assets/dotnet-add-command.png)
+
+We'll copy this command for later use.

```

Comment:
`Let's copy this command and paste it on the command line. In our example, we'll use the WIndows Comand Prompt. Do make sure the working directory is the root of the project we want to .......`